bigbluebutton-client/src/org/bigbluebutton/main/views/AudioSelectionWindow.mxml

```diff
@@ -76,6 +76,12 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
 					vboxListen.percentWidth = 50;
 				}
 				
+				var browser:String = ExternalInterface.call("determineBrowser")[0];
+				if (browser != "Firefox" && browser == "Chrome") {

```

Comment:
If we want to go that route the method we use in the CallManager is `(ExternalInterface.available && ExternalInterface.call("isWebRTCAvailable")`